Flickr/Sources/FlickrAppVCs/SearchResultsVC.swift

```diff
@@ -6,17 +6,183 @@
 //
 
 import Foundation
+import Nuke
 import UIKit
 
+struct Photos: Codable {
+    var photos: PhotoArray
+}
+
+struct PhotoArray: Codable {
+    var photo: [SinglePhoto]
+
+    func getPhotoArray() -> [SinglePhoto] {
+        photo
+    }
+}
+
+struct SinglePhoto: Codable {
+    var id: String
+    var owner: String
+    var secret: String
+    var server: String
+    var title: String
+}
+
 class SearchResultsVC: UIViewController {
+    private var photos: [URL] = []
+    private var searchString: String?
+    private var collectionView: UICollectionView?
+
+    init(searchString: String) {
+        self.searchString = searchString
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func returnImageView(cell: UICollectionViewCell, indexPath: IndexPath) -> [NSLayoutConstraint] {
+        let imageView = UIImageView()
+        imageView.tag = 10
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFill
+        imageView.clipsToBounds = true
+        imageView.sizeToFit()
+        Nuke.loadImage(with: photos[indexPath.row], into: imageView)
+        cell.addSubview(imageView)
+
+        return [
+            imageView.leadingAnchor.constraint(equalTo: cell.leadingAnchor),
+            imageView.trailingAnchor.constraint(equalTo: cell.trailingAnchor),
+            imageView.topAnchor.constraint(equalTo: cell.topAnchor),
+            imageView.bottomAnchor.constraint(equalTo: cell.bottomAnchor)
+        ]
+    }
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = viewBackgroundColor
+        view.backgroundColor = .white
```

Comment:
Please always use R.color to access color assets. THis allows us to easily change the color scheme of the app from a single location.